src/tests/ftest/scripts/main.sh

```diff
@@ -230,8 +230,7 @@ ulimit -n 4096
 launch_args="-jcrisa"
 # processing cores is broken on EL7 currently
 id="$(lsb_release -si)"
-if { [ "$id" = "CentOS" ]                 &&
-     [[ $(lsb_release -s -r) != 7.* ]]; } ||
+if { [ "$id" = "CentOS" ]                 ||
```

Comment:
(lint) The mentioned syntax error was in this if expression. [SC1009]
(lint) Couldn't parse this brace group. Fix to allow more checks. [SC1073]

Reply:
Fixed